### PR TITLE
docs: edit ng serve option doc

### PR DESCRIPTION
**Pull request by jbogarthyde**
Edit doc for ng serve to resolve Issue 
https://github.com/angular/angular-cli/issues/8986

### CODE REVIEW

File: packages/angular/cli/lib/config/schema.json [hunk 1]
```diff
@@ -1063,62 +1063,62 @@
...
             "liveReload": {
               "type": "boolean",
-              "description": "Whether to reload the page on change, using live-reload.",
+              "description": "When true, reload the page on change using live-reload.",
               "default": true
             },
             "publicHost": {
               "type": "string",
-              "description": "Specify the URL that the browser client will use."
+              "description": "The URL that the browser client will use for live reload."
```
Comment (filipesilva) [inline packages/angular/cli/lib/config/schema.json]: `publicHost` is not just about live-reload. @clydin can you offer a better description please?

Comment (clydin) [reply packages/angular/cli/lib/config/schema.json]: `for live reload` -> `to connect to the development server`

Comment (jbogarthyde) [reply packages/angular/cli/lib/config/schema.json]: The issue was that the user didn't relate publicHost to live-reload. How about  this:
"The URL that the browser client will use to connect to the development server for live-reload."

Comment (jbogarthyde) [reply packages/angular/cli/lib/config/schema.json]: @clydin Please check new text

**Comment by jbogarthyde** [comment]
There's no Mark for Merge label - is there something else I have to do to get this merged? @filipesilva

**Comment by filipesilva** [comment]
@jbogarthyde it's just waiting for the caretaker to merge it now, I think. There's nothing else that needs to be done.

**Pull request merged**